core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/SetSearchRequestEvaluator.java

```diff
@@ -47,6 +47,11 @@
 		 * Number of matches to return.
 		 */
 		LIMIT, 
+		
+		/**
+		 * Search for mappings with the specified source tooling id.
+		 */
+		TOOLING_ID
```

Comment:
Rename this to `SOURCE_TOOLING_ID`, please. In case we need a target one as well in the future.

Reply:
Done